.gitignore

```diff
@@ -0,0 +1,10 @@
+.Rproj.user
```

Comment:
Please remove this file from Git tracking.

Gitignore is specific for developer, not package